src/listThirdParties.js

```diff
@@ -0,0 +1,19 @@
+const AWS = require("aws-sdk");
+const dynamodb = require("../libs/dynamodb-lib");
+const tableName = "third_parties";
```

Comment:
I don't see the third_parties table created via serverless.yml file. I created a table Entities. Let's use the same table

And you can user here something like:
const tableName = process.env.ENTITIES_TABLE;